<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeoDataNetcdfImporterDateSelectDialog</class>
 <widget class="QDialog" name="GeoDataNetcdfImporterDateSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Specify Original Date</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="warningLabel">
     <property name="text">
      <string>Automatic recognition of original time failed. Please specify the original date.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
     <item row="0" column="0">
      <widget class="QLabel" name="unitsLabel">
       <property name="text">
        <string>Units:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="unitsValueLabel">
       <property name="text">
        <string>days since 2014-01-01</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="importUnitsAsLabel">
       <property name="text">
        <string>Unit:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="importUnitsAsComboBox">
       <item>
        <property name="text">
         <string>years</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>days</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hours</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>seconds</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>Original time:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDateTimeEdit" name="dateTimeEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="timeZoneLabel">
       <property name="text">
        <string>Time Zone:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTimeZoneSelectComboBox" name="timeZoneComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>buttonBox</zorder>
  <zorder>warningLabel</zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTimeZoneSelectComboBox</class>
   <extends>QComboBox</extends>
   <header>misc/qtimezoneselectcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>importUnitsAsComboBox</tabstop>
  <tabstop>dateTimeEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GeoDataNetcdfImporterDateSelectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GeoDataNetcdfImporterDateSelectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
